torch_uncertainty/routines/regression.py

```diff
@@ -76,7 +79,9 @@ def __init__(
             )
 
         self.model = model
-        self.probabilistic = probabilistic
+        self.dist_family = dist_family
+        self.dist_estimate = dist_estimate
```

Comment:
Why is `self.dist_estimate` here and not in pixel-wise regression?

Reply:
That's a mistake, thanks!